app/views/timed_service_price_policies/_table.html.haml

```diff
@@ -0,0 +1 @@
+= render "time_based_price_policies/table", price_policies: price_policies, url_date: url_date, cancellation: false, minimum_cost: false
```

Comment:
Resolved by the discussion in https://github.com/tablexi/nucore-open/pull/1178#discussion_r129665919